### PR TITLE
chore(*) UPGRADE.md instructions after renaming remote cp to zone cp #2

### PR DESCRIPTION
**Pull request by lobkovilya**
### Summary

follow up for https://github.com/kumahq/kuma/pull/2165

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -6,6 +6,80 @@ with `x.y.z` being the version you are planning to upgrade to.
...
+    New:
+    ```yaml
+    type: ZoneIngress
+    name: zone-ingress
+    networking:
+      address: <ADDRESS>
+      port: <PORT>
+      advertisedAddress: <PUBLIC_ADDRESS>
+      advertisedPort: <PUBLIC_PORT>
+    ```
```
Comment (jakubdyszkiewicz) [inline UPGRADE.md:71]: add

`Note: ZoneIngress resource is a global scoped resource, it's not bound to a Mesh`
`The old Dataplane resource is still supported but it's considered deprecated and will be removed in the next major version of Kuma`

File: UPGRADE.md [hunk 2]
```diff
@@ -6,6 +6,80 @@ with `x.y.z` being the version you are planning to upgrade to.
...
+
+2. `kuma-dp run` command should be updated with a new flag `--proxy-type=ingress`:
+
+    ```sh
+    kuma-dp run \
+      --proxy-type=ingress \
+      --dataplane-token-file=/tmp/zone-ingress-token \
```
Comment (jakubdyszkiewicz) [inline UPGRADE.md:78]: add a step with new token generation

**Review by jakubdyszkiewicz — CHANGES_REQUESTED** [review]

File: UPGRADE.md (hunk 1 above)
Comment (lobkovilya) [reply UPGRADE.md:71]: added. thanks!

File: UPGRADE.md (hunk 2 above)
Comment (lobkovilya) [reply UPGRADE.md:88]: Oh right! that's a pretty important part, thanks!

**Review by jakubdyszkiewicz — APPROVED** [review]

**Pull request merged**